src/components/Table/TableToolbar/TableToolbar.jsx

```diff
@@ -70,6 +70,14 @@ const StyledTableBatchActions = styled(TableBatchActions)`
   }
 `;
 
+const ToolBarResultLabel = styled.label`
```

Comment:
Since we are moving away from styled components these style should be added to a class that is on your label element.